src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -1061,6 +1061,39 @@ static bool address_available(sd_dhcp_server *server, be32_t address) {
         return true;
 }
 
+static int server_get_static_lease(sd_dhcp_server *server, const DHCPRequest *req, DHCPLease **ret) {
```

Comment:
Please add assertions for the arguments.

Reply:
Added `assert()`. Was unsure if it should be `assert(...)` or `assert_return(..., -EINVAL)`. Looking at other functions it seemed to be a mix and couldn't spot any logic in it. So hope `assert()` is correct.

Reply:
In this case, assert() is correct. Thanks.
assert_return() is used only in exposed library functions.